### DISCUSSION TITLE
How to set the error message for a specific rule?

### DISCUSSION THREAD

**Discussion by deanCompRSA** [vueform/vueform#292] [Questions] [answered]
I know that for a custom rule you can set it's error message with the msg

  get msg() {
    return `Custom error: ${this.customError}`
  }
  
  But how do I set a different error message for a none custom rule such as the required rule to say "This field cannot be left empty" or something

**Comment by adamberecz** [comment, marked as answer]
https://vueform.com/docs/validating-elements#custom-error-messages
